src/app/pages/table-page/pagination-table.component.ts

```diff
@@ -1,49 +1,58 @@
 import {
   CUSTOM_ELEMENTS_SCHEMA,
-  Component,
-  ElementRef,
-  ViewChild,
-  inject,
-  Renderer2,
-  AfterViewInit, Input
+  Component
 } from '@angular/core'
-import {TableData} from './table-data'
+import exampleData from './exampleData.json';
+import { CommonModule } from '@angular/common';
 
 @Component({
   selector: 'app-pagination-table',
   template: `
     <h1>Pagination Table</h1>
-    <tds-table vertical-dividers="false" compact-design="false" enable-responsive="false">
+    <tds-table vertical-dividers="false" compact-design="false" responsive="false">
       <tds-table-header>
         <tds-header-cell column-key="truck" column-title="Truck type"></tds-header-cell>
         <tds-header-cell column-key="driver" column-title="Driver name"></tds-header-cell>
         <tds-header-cell column-key="country" column-title="Country"></tds-header-cell>
         <tds-header-cell column-key="mileage" column-title="Mileage" text-align="right"></tds-header-cell>
       </tds-table-header>
-      <tds-table-body #paginationTableBody></tds-table-body>
-      <tds-table-footer default-page="1" pages="4" pagination (tdsPageChange)="paginate($event)"></tds-table-footer>
+      <tds-table-body>
+        <tds-table-body-row *ngFor="let row of tableData">
+          <tds-body-cell cell-key="truck">
+              {{ row.truck }}
+          </tds-body-cell>
+          <tds-body-cell cell-key="driver">
+              {{ row.driver }}
+          </tds-body-cell>
+          <tds-body-cell cell-key="country">
+              {{ row.country }}
+          </tds-body-cell>
+          <tds-body-cell cell-key="mileage">
```

Comment:
The mileage Table column title seems to be right-aligned. Also you can't hover the columns in the pagination Table, only the title (rather than the whole column) gets the hover effect
![image](https://github.com/scania-digital-design-system/tegel-angular-demo/assets/48509966/0a11ccc1-52f1-4f19-a52c-2a6b0facb62d)